src/views/Organisations/Organisation.vue

```diff
@@ -0,0 +1,367 @@
+                        <v-list-item-content><v-list-item-title> Preview record </v-list-item-title></v-list-item-content>
+                      </v-list-item>
+                      <v-list-item
+                        v-if="user().is_curator"
+                        @click="goToEdit(item.fairsharingRecord.id)"
+                      >
+                        <v-list-item-avatar><v-icon>fas fa-pen</v-icon></v-list-item-avatar>
+                        <v-list-item-content><v-list-item-title> Edit record </v-list-item-title></v-list-item-content>
+                      </v-list-item>
+                    </v-list>
+                  </v-menu>
+                </template>
+              </v-data-table>
+              <v-card-text />
+            </v-card>
+          </v-col>
+        </v-row>
+
+        <!-- PREVIEW RECORD -->
+        <v-dialog v-model="showOverlay">
+          <v-btn
+            fab
+            small
+            class="grey--text absolute"
+            @click="hideOverlay()"
+          >
+            <v-icon>fa-times</v-icon>
+          </v-btn>
+
+          <v-card>
+            <Record :target="targetID" />
+          </v-card>
+        </v-dialog>
+
+        <v-fade-transition>
+          <v-overlay
+            v-if="loading"
+            :absolute="false"
+            opacity="0.8"
+          >
+            <loaders />
+          </v-overlay>
+        </v-fade-transition>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import GraphClient from "@/components/GraphClient/GraphClient.js"
+import Icon from "@/components/Icon";
+import Loaders from "@/components/Navigation/Loaders";
+import NotFound from "@/views/Errors/404"
+import Record from "@/views/Records/Record";
+import StatusPills from "@/components/Users/Profiles/Private/StatusPills";
+import getOrganisationQuery from "@/components/GraphClient/queries/Organisations/getOrganisation.json"
+import { cleanString, formatList } from "@/utils/stringUtils"
+import {mapState} from "vuex";
+
+let graphClient = new GraphClient();
+
+export default {
+  name: "Organisation",
+  components: { NotFound, Icon, Loaders, StatusPills, Record },
+  mixins: [cleanString, formatList],
+  data: () => {
+    return {
+      error: true,
+      organisation: {},
+      loading: false,
+      perPage: 10,
+      footer: {'items-per-page-options': [10]},
+      showOverlay: false,
+      targetID: null,
+      headers: [
+        {text: 'Name', value: 'name', align: 'center'},
+        {text: 'Status', value: 'status', align: 'center'},
+        {text: 'Relation', value: 'relation', align: 'center'},
+        {text: 'Grant', value: 'grant', align: 'center'},
+        {text: 'Actions', value: 'actions', align: 'center', sortable: false}
+      ]
+    }
+  },
+  computed: {
+    ...mapState('users', ['user']),
+    currentRoute() {
+      return this.target || this.$route.params['id'];
```

Comment:
not sure what this.target refers to, it's never used nor declared. Something you copy pasted from another component ?
```suggestion
      return this.$route.params['id'];
```

Reply:
Isn't it related to the record preview overlay?